src/greenlight/step.clj

```diff
@@ -272,6 +272,15 @@
     (fn? (::title step)) (update ::title #(% ctx))))
 
 
+(defmulti report->outcome :type)
+
+(defmethod report->outcome :pass [_] :greenlight.assert/pass)
+(defmethod report->outcome :fail [_] :greenlight.assert/fail)
+(defmethod report->outcome :error [_] :greenlight.assert/error)
+(defmethod report->outcome :matcher-combinators/mismatch [_] :greenlight.assert/fail)
```

Comment:
:-1: on coupling directly to other library types. This isn't a lot of code to ask consumers to add if they want to integrate, and I can foresee confusion about why some things work automatically and others don't. It would be good to drop some examples like this in the README or a more custom integration doc, though.